src/components/record/CityList.tsx

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+import { COUNTRY_CODE_TO_FLAG } from "@/constants/countryMapping";
+import type { RecordResponse } from "@/types/record";
+
+interface CityListProps {
+  filteredRegions: RecordResponse["data"]["regions"];
+}
+
+const getCountryFlagByCode = (countryCode: string): string => COUNTRY_CODE_TO_FLAG[countryCode] || "🌍";
+
+export function CityList({ filteredRegions }: CityListProps) {
+  const router = useRouter();
+
+  return (
+    <div className="flex flex-col gap-[30px] pb-8">
+      {filteredRegions.map((region) => (
+        <div key={`${region.regionName}-${region.cities[0]?.countryCode || ""}`} className="flex flex-col gap-3">
+          <div className="text-white text-base font-medium">
+            {getCountryFlagByCode(region.cities[0]?.countryCode || "")} {region.regionName}
```

Comment:
```suggestion
            {getCountryFlagByCode(region.cities[0]?.countryCode)} {region.regionName}
```
앞에서 이미 옵셔널체이닝으로 예외처리를 하고있어서, 
현재 getCountryFlagByCode에서는 falsy 여부만 체크하기때문에 불필요한 이중 처리로 보여요

Reply:
https://github.com/depromeet/17th-team1-client/pull/57/commits/9e65727418abe96ca1f91fbb509fc7b32965b79b

그렇네요 꼼꼼하게 봐주셔서 감사합니다 😊